src/runtime/virtcontainers/README.md

```diff
@@ -232,7 +232,7 @@ is also supported in Kata. Pass-through happens separated by adapter and
 domain, i.e. a passable VFIO device has one or multiple adapter-domain
 combinations.
 
-1. You must follow the [kernel documentation for preparing VFIO-AP passthrough](https://www.kernel.org/doc/html/latest/s390/vfio-ap.html).
+1. You must follow the [kernel documentation for preparing VFIO-AP passthrough](https://www.kernel.org/doc/Documentation/s390/vfio-ap.txt).
```

Comment:
This one also being addressed in https://github.com/kata-containers/kata-containers/pull/8079 . Can I close 8079 in favor of this one?

Reply:
@wainersm you could close it if you want, if you decide to merge it just please let me know here so I can remove this bit.